src/documents/admin.py

```diff
@@ -70,9 +70,13 @@ def created_(self, obj):
     created_.short_description = "Created"
 
     def thumbnail(self, obj):
+        if settings.FORCE_SCRIPT_NAME:
+            _src="{}/fetch/thumb/{}".format(settings.FORCE_SCRIPT_NAME,obj.id)
+        else:
+            _src="/fetch/thumb/{}".format(obj.id)
         png_img = self._html_tag(
             "img",
-            src="/fetch/thumb/{}".format(obj.id),
+            src=_src,
```

Comment:
Please don't prefix local variables with an underscore like this.  It makes sense for instance variables or class methods (as the underscore denotes privacy), but in a local variable it means nothing.